### DISCUSSION TITLE
This is incredible... how hard would it be to modify for use with ipfire?

### DISCUSSION THREAD

**Discussion by hagfelsh** [pfelk/pfelk#523] [General]
The implementation of pfelk is just perfect...exactly what I'm looking for!  Except... I'm not familiar with the *senses enough to guess at how much work it would be to make pfelk work with ipfire.

Does anyone here know enough about it to offer their insight?

**Comment by a3ilson** [comment]
@hagfelsh 

Incorporating within the existing pfelk config should be easily doable.


1) Configured pipelines.yml for the ipfire logs
- Reference :https://www.elastic.co/guide/en/logstash/current/multiple-pipelines.html

2) You'll need to create the necessary configuration files to interrupt and enrich the ipfire logs

3) Build out visualization with indices and visualization for new data (logs)

Also take a look at #489
